### PR TITLE
docs: pr template

### PR DESCRIPTION
**Pull request by emilyjablonski**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## How Can This Be Tested/Reviewed?
+
+Provide instructions so we can review.
+
+Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
```
Comment (kramduckner) [inline pull_request_template.md:17]: are the style guidelines the style conventions in the README?

**Review by kramduckner — APPROVED** [review]

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] I have performed a self-review of my own code
+- [ ] I have reviewed the changes in a desktop view
+- [ ] I have reviewed the changes in a mobile view
+- [ ] I have made any corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have exported any new pieces added to ui-components
+- [ ] I have documented this change in the changelog, and any breaking changes are well described
+
```
Comment (ludtkemorgan) [inline pull_request_template.md:28]: I'm wondering if we should add a check for storybook as well? Something like `I have added or updated the corresponding storybook if necessary`

**Review by ludtkemorgan — APPROVED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (emilyjablonski) [reply pull_request_template.md:17]: yep! i also have an incoming pr to update the readme with some more detail about picking up issues / contributing

File: pull_request_template.md (hunk 2 above)
Comment (emilyjablonski) [reply pull_request_template.md:29]: Added!

**Pull request merged**